CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing to `electron-installer-debian`
+
+`electron-installer-debian` is a community-driven project. As such, we welcome and encourage all
+sorts of contributions. They include, but are not limited to:
+
+- Constructive feedback
+- [Questions about usage](https://github.com/unindented/electron-installer-debian/blob/master/SUPPORT.md)
+- [Bug reports / technical issues](#before-opening-bug-reportstechnical-issues)
+- Documentation changes
+- Feature requests
+- [Pull requests](#filing-pull-requests)
+
+We strongly suggest that before filing an issue, you search through the existing issues to see
+if it has already been filed by someone else.
```

Comment:
We should mention that people need to search open and closed issues. Many times, they don't and we end up with duplicates. As an example, issues #107 #118 and #128 are all the same thing.